### ISSUE TITLE
Main.py fails to import call_peaks.py, prints usage for peaksmodule.cc instead

### ISSUE THREAD

**Issue by lee-van** [YeoLab/clipper#101]
I am running clipper version 2.1.0 commit 75160fd556fe73dbbcc6c0a806c4e672073a6b00 from the .yml conda environment included in this repo.

The main.py called by the clipper wrapper is failing at this step:
`from clipper.src.call_peak import call_peaks`

Running clipper, with or without any parameters, outputs a usage message from peaksmodule.cc: 
 ```
[options]
file_in is a file with a list of the length of aligned reads (only the part that aligns)
Options
 -L <int>   Effective Gene Length
 -r <int>   # of iterations
 -f <int>   input file containing read lengths
 -a <float>   B-H FDR cutoff, default(.05) ... gets very slow as alpha gets smaller
 -T 1/0 default(0) print running time statistics
Output:
```

If I comment out `from clipper.src.call_peak import call_peaks`, I can at least have clipper print its normal usage but fail to call peaks. I suspect there is a bug in call_peaks.py

**Comment by kerenzhou062** [comment]
Yeah! I have the same problem! Very annoying with this bug! Spending a whole day still can't solve it. Depressed.

**Comment by QuanLi-bio** [comment]
I have the same problem too! But didn't have any solutions to solve it.  I was trapped for 2 days.
